src/esp32c3/include/soc/README.txt

```diff
@@ -0,0 +1 @@
+Generated code
```

Comment:
do we really need this file?

Reply:
Right, I've already remove it locally. Just seeking another place for the notice that soc/ dir contains the generated code (took from IDF now). Perhaps in the root README or somewhere. not critical though